src/Symfony/Bridge/Doctrine/composer.json

```diff
@@ -43,7 +43,7 @@
         "symfony/uid": "^6.4|^7.0",
         "symfony/validator": "^6.4|^7.0",
         "symfony/var-dumper": "^6.4|^7.0",
-        "doctrine/collections": "^1.0|^2.0",
+        "doctrine/collections": "^1.8|^2.0",
```

Comment:
Do we need to bump 2.0 too or how is the release process fur doctrine/collections ?

Reply:
1.8 and 2.0 have the same feature set.